Source/JavaScriptCore/yarr/YarrParser.h

```diff
@@ -1832,7 +1832,7 @@ class Parser {
     }
 
     template<UnicodeParseContext context>
-    int tryConsumeUnicodeEscape()
+    char32_t tryConsumeUnicodeEscape()
```

Comment:
Should change this back to `int` for now for consistency with the rest of the file.

Reply:
Or maybe I should go the other way and convert more fully to `char32_t` here and add an error value instead of `-1`.

Reply:
I did go the other way.